pkg/vm/cli/cli_test.go

```diff
@@ -256,6 +262,9 @@ func TestLoad(t *testing.T) {
 		require.NoError(t, ioutil.WriteFile(filenameErr, append([]byte{1, 2, 3, 4}, rawNef...), os.ModePerm))
 		notExists := filepath.Join(tmpDir, "notexists.json")
 
+		filename = "'" + "'"
+		filenameErr = "'" + "'"
```

Comment:
So both are `''`? Why? Maybe it's `manifestFile` that needs to be wrapped here?

Reply:
Why? Filenames are quoted in order to avoid path delimiter disappearence inside `e.runProg`. `e.runProg` writes all its arguments to input, and if the path is not escaped properly (or not quoted), then all backslashes will be removed from the path.

Reply:
You're right, manifest file also should be wrapped.